server/container/guice/cassandra-rabbitmq-guice/src/test/java/org/apache/james/WithEncryptedSwiftExtension.java

```diff
@@ -30,14 +30,14 @@
 import org.junit.jupiter.api.extension.ParameterResolutionException;
 import org.junit.jupiter.api.extension.ParameterResolver;
 
-public class WithEncryptedSwiftExtension implements CassandraRabbitMQJamesServerTest, BeforeAllCallback, AfterAllCallback, BeforeEachCallback, AfterEachCallback, ParameterResolver {
+public class WithEncryptedSwiftExtension implements CassandraRabbitMQJamesServerFixture, BeforeAllCallback, AfterAllCallback, BeforeEachCallback, AfterEachCallback, ParameterResolver {
```

Comment:
this `CassandraRabbitMQJamesServerFixture` here is not related, you can make the `CassandraRabbitMQJamesServerFixture.baseExtensionBuilder()` become a static method.